wiki/Music_theory/Beat/es.md

```diff
@@ -0,0 +1,22 @@
+---
+outdated_translation: true
```

Comment:
please remove this if you redo the translation according to https://github.com/ppy/osu-wiki/blob/9adf17368908ee752d1fc0c6c54212c0bb5a8de5/wiki/Music_theory/Beat/en.md